src/index.ts

```diff
@@ -87,6 +85,7 @@ export interface GetSubscriptionsCallback {
 }
 
 export interface GetTopicsOptions extends Options {}
+export type GetTopicsResponse = [Topic[], object];
```

Comment:
I'm actually working a PR that overhauls some of the types, but I think the signature for all the lists is incorrect. It can either be one of the following

```typescript
type AllTopics = [Topic[]];
type TopicsPage = [Topic[], {}|null, google.pubsub.v1.IListTopicsResponse]
export type GetTopicsResponse = AllTopics|TopicsPage; 
```